code/modules/clothing/suits/marine_armor/_marine_armor.dm

```diff
@@ -254,7 +254,7 @@
 	desc = "A standard Colonial Marines M2 Pattern Chestplate. Protects the chest from ballistic rounds, bladed objects and accidents. It has a small leather pouch strapped to it for limited storage."
 	icon_state = "mp_armor"
 	armor_melee = CLOTHING_ARMOR_MEDIUMHIGH
-	armor_bullet = CLOTHING_ARMOR_LOW
+	armor_bullet = CLOTHING_ARMOR_MEDIUMHIGH
```

Comment:
this is an insane jump on armor values . at that point why use riot armor if the base just has 5 less ??? 
i think MEDIUMLOW or MEDIUM would be better

Reply:
remember this armor has MEDIUMHIGH MELEE and very little slowdown

Reply:
I'll amend it to medium. Unless you want to split MP and Officer armor.